terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,78 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "public-data" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "company-appointments"
```

Comment:
Think this needs to be company-appointments-api (same format as in certified-copies)